zenduty/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Use the Zenduty integration to send Datadog alerts to the right team, notify them as per on-call schedules, and help them remediate and resolve incidents with speed.
+
+Connect Zenduty with Datadog in order to:
+- Trigger and resolve incidents, get alerts for them, and track issues from Datadog
+- Deploy on-call schedules, escalation policies, incident playbooks, postmortems, and detailed analytics
+- Use Alert Rules to custom route specific Datadog alerts to certain users or teams, write suppression rules, and automatically add notes, responders, and incident tasks
+
+## Setup
+
+### Zenduty
+Follow the following steps inside Zenduty:
+
+1. Go to **Teams** and click on the team you want to add the integration to.
+ 
+2. Navigate to **Services**. Create a new service or select an existing one.
+ 
+3. Go to **Integrations** and then **Add New Integration**. Give the integration a name, and select the application **Datadog** from the dropdown menu.
+
+4. Go to **Configure** under your integrations, and copy the generated Datadog webhook URL.
+
+### Datadog
+
+5. From the sidebar, go to **Integrations**. Search for **Webhooks** from this page, and click the add button.
```

Comment:
If you are referring to the webhook page in Datadog, it would be helpful to add a hyperlink: https://app.datadoghq.com/integrations/webhooks?search=webhook

Reply:
Added!